Zone.UmbracoPersonalisationGroups/Configuration/PersonalisationGroupsConfig.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Zone.UmbracoPersonalisationGroups.Configuration
+{
+    using System.Configuration;
+
+    /// <summary>
+    /// Configuration for personalisation groups
+    /// </summary>
+    public class PersonalisationGroupsConfig
```

Comment:
Created this for the purpose of testing (inspired by ModelsBuilder), but does also provide a nice wrapper for configuration so might be worth fleshing out all the other properties here.